packages/playwright-core/src/server/har/harTracer.ts

```diff
@@ -517,13 +508,19 @@ export class HarTracer {
   }
 
   async flush() {
-    await Promise.all(this._barrierPromises);
+    const promises = [...this._barrierPromises];
+    this._barrierPromises.clear();
+    await Promise.race([
+      Promise.all(promises),
+      new Promise(f => setTimeout(f, 1000))
```

Comment:
I wonder whether 1 second is too low on CI to fetch response bodies?

Reply:
It is on the server side and the content should be cached, so I would say should be plenty?